ui/src/app/views/workspace-wrapper/component.html

```diff
@@ -0,0 +1,21 @@
+<app-workspace-nav-bar *ngIf="displayNavBar"
```

Comment:
quick ELI5 - what is workspace-wrapper as it relates to nav bar?  why aren't these all one thing?

Reply:
The wrapper contains the nav bar, the workspace-level modals, and crucially, the router outlet. We may decide to do things differently once the react conversion is further along, but for the time being I needed an overarching angular component to control the modals and pass the functions to display them in.